frontend/src/views/WasteActionsPage/WasteActionsHome.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div id="wasteactions-home" class="text-left">
+    <BreadcrumbsNav />
+    <div v-if="loading" class="mt-8">
+      <v-progress-circular indeterminate></v-progress-circular>
+    </div>
+    <div v-if="!loading && wasteactions.length > 0">
+      <h1 class="fr-h1 mb-4">
+        Explorez et mettez en avant les actions de lutte contre le gaspillage
```

Comment:
```suggestion
        Lutter contre le gaspillage alimentaire
```

les actions mettre en avant sont que applicable quand on commence à travailler sur les fonctionnalités pour les cantines. Je propose une action plus directe (`lutter`) par un preference perso, je te laisse decider si c'est mieux ou plutôt `Explorer les actions de lutte contre le gaspillage`